nixos/modules/security/google_oslogin.nix

```diff
@@ -50,6 +50,9 @@ in
     systemd.packages = [ package ];
     systemd.timers.google-oslogin-cache.wantedBy = [ "timers.target" ];
 
+    # Without network this will fail to fetch users which causes sudo not to work.
```

Comment:
How does this fail? Is it crashing (without being automatically restarted) because the network stack isn't up, or does it actively need an active network connection?

https://systemd.io/NETWORK_ONLINE/ says for `network.target`:

> network.target indicates that the network management stack has been started. *Ordering after it it has little meaning during start-up: whether any network interfaces are already configured when it is reached is not defined*.

Are you sure this fixes the issue in all cases?

Reply:
It fails to fetch google os metadata. network.target was enough here in my testing. There might be still cases where there are some temporary network failures but than the systemd timer will eventually refetch the cache.

Reply:
What was the error message ?

Reply:
As per these docs, `network.target` is still wrong, and it might just work fine with `network-online.target`. Can you change it to that?